### ISSUE TITLE
apache plugin can't parse owncloud.conf

### ISSUE THREAD

**Issue by hydrargyrum** [letsencrypt/letsencrypt#1264]
When running

    ./letsencrypt-auto -d DOMAIN.TLD  --apache auth

I get

    The apache plugin is not working; there may be problems with your existing configuration.
    The error was: PluginError(('There has been an error in parsing the file (%s): %s', u'/etc/apache2/conf-enabled/owncloud.conf', u'Syntax error'),)

Here's my owncloud.conf (straight from the debian owncloud package):

```
Alias /owncloud /usr/share/owncloud

<Directory /usr/share/owncloud/>
    Options +FollowSymLinks
    AllowOverride All
    <IfVersion < 2.3>
        order allow,deny
        allow from all
    </IfVersion>
    <IfVersion >= 2.3>
        Require all granted
    </IfVersion>
</Directory>
```

**Comment by xamnesiax-sx** [comment]
Could it be that the regex is hitting the "<" and ">" symbols and screwing up? Seems like a possibility, assuming you use RE for the config parsing.

**Comment by hydrargyrum** [comment]
What do you mean? LE-auto seems to use augeas for parsing the configuration.

**Comment by chriscroome** [comment]
It appears to me that augeas was fixed to work with the following syntax https://github.com/hercules-team/augeas/issues/154

```apache
    <IfVersion >= 2.3>
        Require all granted
    </IfVersion>
```

But was it fixed to work with this?

```apache
    <IfVersion < 2.3>
        order allow,deny
        allow from all
    </IfVersion>
```

**Comment by domcleal** [comment]
@hydrargyrum which OS + version are you on?  Can you check the version of Augeas?

**Comment by hydrargyrum** [comment]
I'm using debian sid, with libaugeas0 1.2.

**Comment by domcleal** [comment]
Yeah, that's missing the fix that @chriscroome mentions, which is in Augeas 1.3.0+.  I don't know why the maintainer hasn't updated it in unstable yet.

**Comment by pde** [comment]
If as @domcleal says this was fixed in Augeas 1.3.0+, this issue should be resolved when #1291 lands.  But let's confirm that and add a test case if we can.

**Comment by pde** [comment]
This was fixed by #1291.  Thanks team!

**Comment by jojom4n** [comment]
It seems it has not been fixed, to be honest, as reported in letsencrypt/letsencrypt#1734
